### ISSUE TITLE
Did you experiment with images?

### ISSUE THREAD

**Issue by Jumabek** [enamul-haque/Byte2Vec#1]
Hi I saw jpeg extensions in feature generator code. 

Do byte2vec features work well with images as well?

**Comment by enamul-haque** [comment]
Yes I did. The model works for any type of files. You can include a type and files in the list if those are not in the provided list to generate your own model.

**Comment by Jumabek** [comment]
@enamul-haque thanks a lot for your prompt response.

Do I need to train a model from images?

Or is the model trained from govdocs_files will work well for the extracting image features?

**Comment by enamul-haque** [comment]
Byte2vec was designed to work with multiple file types. If you want to train it using only image, you need to modify the code accordingly. However, If you want to use the images from govdocs as your training image, then remove all other file extensions from the code. I would suggest using large number of image file types to train byte2vec for your purpose. You can include your own image examples along with the govdocs samples.
